test/e2e/specs/editor/various/preview.spec.js

```diff
@@ -0,0 +1,362 @@
+/**
+ * WordPress dependencies
+ */
+const { test, expect } = require( '@wordpress/e2e-test-utils-playwright' );
+
+test.use( {
+	previewUtils: async ( { page }, use ) => {
+		await use( new PreviewUtils( { page } ) );
+	},
+} );
+
+test.describe( 'Preview', () => {
+	test.beforeEach( async ( { admin } ) => {
+		await admin.createNewPost();
+	} );
+
+	test( 'should open a preview window for a new post', async ( {
+		editor,
+		page,
+		pageUtils,
+		previewUtils,
+		requestUtils,
+	} ) => {
+		const editorPage = page;
+
+		// Disabled until content present.
+		const isPreviewDisabled = await editorPage
+			.locator( 'role=button[name="Preview"]' )
+			.isDisabled();
+		expect( isPreviewDisabled ).toEqual( true );
```

Comment:
We can do it in Playwright-style

```suggestion
		// Disabled until content present.
		await expect(
			editorPage.locator( 'role=button[name="Preview"i]' )
		).toBeDisabled();
```